IFC4x3/PropertyEnumerations/a/PEnum_AntennaRadiationPattern.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocPropertyEnumeration xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="PEnum_AntennaRadiationPattern" Name="PEnum_AntennaRadiationPattern" UniqueId="fdfd847b-d2d3-4773-8231-11e10d65de04">
+	<Documentation>Enumeration of antenna radiation patterns.</Documentation>
+	<Constants>
+		<DocPropertyConstant xsi:nil="true" href="DIRECTIONAL_3i7a7qyYDCX9BH5IjBfVd2" />
+		<DocPropertyConstant xsi:nil="true" href="OMNIDIRECTIONAL_2hx94KqpT23xSOQXkUgcjb" />
+		<DocPropertyConstant xsi:nil="true" href="OTHER_0SAtxxJj15wQA15w2MsIV" />
+		<DocPropertyConstant xsi:nil="true" href="NOTKNOWN_0_n254Fb10hBW_0_XIj7Qe" />
+		<DocPropertyConstant xsi:nil="true" href="UNSET_01y959UgrAlvv6_W__S9Ap" />
```

Comment:
do we think this is sufficient? from my googling i found:

1. Omni directional
2. pencil beam
3. fan beam
4. shaped beam